packages/cli/commands/auth.js

```diff
@@ -158,18 +158,27 @@ exports.handler = async options => {
 };
 
 exports.builder = yargs => {
-  yargs.positional('type', {
-    describe: i18n(`${i18nKey}.positionals.type.describe`),
-    type: 'string',
-    choices: [
-      `${PERSONAL_ACCESS_KEY_AUTH_METHOD.value}`,
-      `${OAUTH_AUTH_METHOD.value}`,
-      `${API_KEY_AUTH_METHOD.value}`,
-    ],
-    default: PERSONAL_ACCESS_KEY_AUTH_METHOD.value,
-    defaultDescription: i18n(`${i18nKey}.positionals.type.defaultDescription`, {
-      authMethod: PERSONAL_ACCESS_KEY_AUTH_METHOD.value,
-    }),
+  yargs.options({
```

Comment:
This is a breaking change for the "type" arg. We're moving towards using options for all of our args, but we'll need to keep this backwards compatible. We can support the `--type` option, but we'll still need to support the "type" positional

Reply:
Hmm, might be too verbose to keep the [--type] option there alongside the positional in the `command` line, unless we keep it hidden until its not needed?